changelog/5057.bugfix.rst

```diff
@@ -0,0 +1 @@
+Addressing ISSUE-2760 to resolve compatibility issues with Microsoft Bot Framework Emulator.
```

Comment:
```suggestion
Fixed compatibility issue with Microsoft Bot Framework Emulator if ``service_url`` lacked a trailing ``/``.
```

Reply:
@justinas-b can you please commit this change as I don't have push access to your fork?